lib/basedriver/helpers.js

```diff
@@ -357,6 +357,27 @@ function duplicateKeys (input, firstKey, secondKey) {
   return input;
 }
 
+/**
+ * Takes a desired capability and tries to JSON.parse it as an array,
+ * and either returns the parsed array or a singleton array.
+ *
+ * @param {any} cap A desired capability
+ */
+function parseArray (cap) {
+  let parsedCaps;
```

Comment:
I'd rather select some more generic var name

Reply:
or, like @imurchie mentioned, rename the function to reflect the fact it is used for capabilities